packages/docs/components/page-configs/va-app-bar/api-options.ts

```diff
@@ -0,0 +1,18 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
Demos are cool looking. Can we use them maybe?
![image](https://user-images.githubusercontent.com/5394573/102542606-1c263b00-40c3-11eb-84c5-eed6c9340c0d.png)